lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -96,7 +100,8 @@ def test_process_analytics_answer_dist(self):
                 "module_id": "i4x://A/B/problem/f3ed0ba7f89445ee9a83541e1fc8a2f2",
                 "part_id": "i4x-A-B-problem-f3ed0ba7f89445ee9a83541e1fc8a2f2_2_1",
                 "correct": False,
-                "count": 7,
+                "first_response_count": 7,
+                "final_response_count": 2,
```

Comment:
Dylan is calling it last instead of final now: https://github.com/Stanford-Online/edx-analytics-data-api-client/commit/754535d1705a7e06b1bb37dc92aad8cf8313a735